### ISSUE TITLE
404 error page shown on clicking 'Open in Connect' link for booking Manager.

### ISSUE THREAD

**Issue by sandhiyakavi** [topcoder-platform/taas-app#67]
Description:

'404 error page' is shown on clicking 'Open in Connect' link for booking manager if he/she is not a member of the project.

Steps to reproduce:

1. Create a project  with TonyJ account.
2. Access the same project in Taas app with Booking Manager account.(should not be a member of the project in Connect)
3. Click the 'Open in Connect' from ellipses icon/Team details page.

Actual result:
Getting '404 error page' as booking Manager is not able to access project in Connect app.

@maxceem @nkumar-topcoder Is there a way to handle this error? It might cause confusion to the user as the Booking Manager is able to access the project in Taas app and not in Connect app?

![image](https://user-images.githubusercontent.com/55895538/106094539-88419780-6175-11eb-8eb4-d1d99d2981ea.png)

**Comment by SathyaJayabal** [comment]
So the booking manager has access to all projects in Taas, but may not have access to them in connect. In such acses when they click on open in connect link form taas, we get the above scenario.

**Comment by maxceem** [comment]
Good catch @sandhiyakavi. This is as per current business requirements. 
@wdprice is there anything we can do in such situations?

**Comment by nkumar-topcoder** [comment]
@maxceem @sandhiyakavi Open In **connect** should be enabled only for limited set role. further discussion is needed on this  cc @wdprice
